deploy/openj9-chart/README.md

```diff
@@ -0,0 +1,106 @@
+# OpenJ9 Helm Chart
+
+## Introduction
+
+Eclipse OpenJ9 is an independent implementation of a Java Virtual Machine.
+
+The OpenJ9 Chart allows you to deploy and manage java applications running on OpenJ9 into OKD or OpenShift clusters. You can also perform Day-2 operations such as rolling update your application, enabling JITServer technology for continued performance enhancements or gathering JVM dumps using the operator.
+
+## Resources Required
+
+### System
+
+* CPU Requested : 1 CPU
+* Memory Requested : 256 MB
+
+### Storage
+
+* Currently no storage or volume needed.
+
+## Chart Details
+
+* Install one `deployment` running Adopt OpenJ9 image
```

Comment:
```suggestion
* Install one `deployment` running AdoptOpenJDK OpenJ9 image
```

Does this have to be an Adopt build?  Is any OpenJ9 build from any builder sufficient?

Reply:
No it doesn't have to be an Adopt build. The image could be from anyone as long as it includes OpenJ9. The image location can be changed here (line 22 in `/deploy/openj9-chart/values.yaml`). 
https://github.com/eclipse/openj9/pull/10606/files#diff-44da2148dd4982f2a9acb128fe6227a4R22